### PR TITLE
added gaurav

### PR DESCRIPTION
**Pull request by yednapg**

### CODE REVIEW

**Comment by sampoder** [comment]
hey @yednapg! current policy is that `hackclub.com` domains aren't for personal usage - how about a `dino.icu` domain?

**Review by sampoder — CHANGES_REQUESTED** [review]
^

**Comment by yednapg** [comment]
I think I've requested for `dino.icu`. Isn't @sampoder?

**Comment by sampoder** [comment]
My bad! I missed that - I guess that's what happens when you review too many PRs at once.

File: dino.icu.yaml [hunk 1]
```diff
@@ -100,6 +100,11 @@ emojile: # wordle with slack emojis - https://github.com/yodalightsabr/emojile
   ttl: 1
   type: CNAME
   value: yodalightsabr.github.io.
+  
+gaurav: # gaurav website
+  ttl: 1
+  type: CNAME
+  value: yednapg.vercel.app.
```
Comment (sampoder) [inline dino.icu.yaml:107]:
```suggestion
  value: cname.vercel-dns.com.
```

You'll also need to add the Vercel verification TXT record to `_vercel`

**Comment by yednapg** [comment]
Hey! closing this PR, I just bought my own domain pandeygaurav.com! Thanks! 🙌

**Comment by sampoder** [comment]
hack yeah!!

**Pull request closed without merging**